app/renderer/js/main.js

```diff
@@ -152,6 +158,34 @@ class ServerManagerView {
 		this.toggleSidebar(showSidebar);
 	}
 
+	initPresetOrgs() {
```

Comment:
Ah okay. I'd not seen `async` used anywhere so far, I just assumed we didn't like it in the codebase for some reason. I'll change it.

Reply:
It not visible in the codebase because when most of the stuff was written `async` wasn't supported. Try using whenever possible it will simplify the code significantly. The `then` changing gets hard to follow.